components/console-backend-service/internal/domain/k8s/namespace_service.go

```diff
@@ -16,19 +17,25 @@ import (
 )
 
 type namespaceService struct {
-	informer cache.SharedIndexInformer
-	client   corev1.CoreV1Interface
+	namespacesInformer cache.SharedIndexInformer
```

Comment:
I like the previous version `informer` more. You're not going to have more informers here.

Reply:
we had two informers in one of our versions, and this name stayed here after we removed it ;) will change it 👍